<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Scheduling</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon/logo.png</normaloff>:/icon/logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="sideBar" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>800</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="sideBarWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>99</width>
          <height>800</height>
         </size>
        </property>
        <widget class="QToolButton" name="calendarBtn">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>250</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/Calendar_icon2.png</normaloff>:/icon/Calendar_icon2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QToolButton" name="todoBtn">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>370</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/Todo_icon5.png</normaloff>:/icon/Todo_icon5.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
        </widget>
        <widget class="QToolButton" name="memoBtn">
         <property name="geometry">
          <rect>
           <x>15</x>
           <y>490</y>
           <width>70</width>
           <height>70</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/Memo_icon2.png</normaloff>:/icon/Memo_icon2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>55</width>
           <height>55</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="settingBtn">
         <property name="geometry">
          <rect>
           <x>31</x>
           <y>735</y>
           <width>37</width>
           <height>33</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="splitLine" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>700</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1300</width>
       <height>800</height>
      </size>
     </property>
     <widget class="QWidget" name="calendarPage">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="CalendarWidget" name="calendarWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>800</height>
          </size>
         </property>
         <widget class="QLabel" name="mottoLabel">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>600</y>
            <width>400</width>
            <height>140</height>
           </rect>
          </property>
          <property name="text">
           <string>Motto</string>
          </property>
         </widget>
         <widget class="QLabel" name="celebrityLabel">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>740</y>
            <width>411</width>
            <height>35</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="empty">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>600</y>
            <width>451</width>
            <height>181</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <zorder>empty</zorder>
         <zorder>mottoLabel</zorder>
         <zorder>celebrityLabel</zorder>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="eventWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>800</height>
          </size>
         </property>
         <widget class="QListWidget" name="eventlistWidget">
          <property name="geometry">
           <rect>
            <x>25</x>
            <y>25</y>
            <width>800</width>
            <height>750</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Cambria Math</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::IgnoreAction</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="addBtn">
          <property name="geometry">
           <rect>
            <x>710</x>
            <y>670</y>
            <width>80</width>
            <height>80</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icon/add.png</normaloff>:/icon/add.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>55</width>
            <height>55</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="calendarFullPage"/>
     <widget class="QWidget" name="todoPage">
      <widget class="TodoWidget" name="todoWidget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1300</width>
         <height>800</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="memoPage">
      <widget class="MemoWidget" name="memoWidget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1300</width>
         <height>800</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CalendarWidget</class>
   <extends>QWidget</extends>
   <header location="global">calendarwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TodoWidget</class>
   <extends>QWidget</extends>
   <header location="global">todowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MemoWidget</class>
   <extends>QWidget</extends>
   <header location="global">memowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
